packages/kbn-shared-ux-components/src/page_template/solution_nav/solution_nav.scss

```diff
@@ -0,0 +1,30 @@
+$euiSideNavEmphasizedBackgroundColor: transparentize($euiColorLightShade, .7);
+@import '@elastic/eui/src/components/side_nav/mixins';
```

Comment:
I don't think this should be imported like this. Is it acceptable to just copy the relevant styling as js constants and use them here?

Reply:
The mixin was created by EUI specifically to be used in multiple Elastic products, we don't want to duplicate styles because then they'll get out of sync. This method of importing has been fine and eventually will get converted to a JS hook. So it's ok for now.